Wrappers/Python/test/test_DataContainer.py

```diff
@@ -91,6 +91,12 @@ def testGb_creation_nocopy(self):
         ds1 = ds.clone()
         self.assertNotEqual(aid(ds.as_array()), aid(ds1.as_array()))
 
+    def test_ndim(self):
+
+        x_np = numpy.arange(0, 60).reshape(3,4,5)
+        x_cil = DataContainer(x_np)
+        self.assertEqual(x_np.ndim, x_cil.ndim)
```

Comment:
I think this should test `self.assertEqual(x_cil.ndim, 3)`

Reply:
Fixed